elements/pfe-clipboard/demo/index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+  <title>PatternFly Element | pfe-clipboard Demo</title>
+
+  <noscript>
+    <link href="../../pfelement/dist/pfelement-noscript.min.css" rel="stylesheet">
+  </noscript>
+
+  <link href="../../pfelement/dist/pfelement.min.css" rel="stylesheet">
+
+  <!-- Stylesheets for testing light DOM styles.
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.3.1/css/bootstrap-reboot.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/typebase.css/0.5.0/typebase.css">
+    -->
+
+  <link rel="stylesheet" href="../../pfe-styles/dist/pfe-base.css" />
+  <link rel="stylesheet" href="../../pfe-styles/dist/pfe-context.css" />
+  <link rel="stylesheet" href="../../pfe-styles/dist/pfe-layouts.css" />
+
+  <!-- uncomment the es5-adapter if you're using the umd version -->
+  <script src="/examples/node_modules/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js"></script>
+
+  <!-- Use webcomponents-loader when you are adding support for more modern browsers -->
+  <!-- <script src="/examples/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script> -->
+
+  <!-- IE11 test: use bundle + require with umd files -->
+  <!-- Use webcomponents-bundle when supporting much older browsers like IE11 -->
+  <script src="/examples/node_modules/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+
+  <!-- Tests require pulling in the UMD version of the files -->
+  <script src="/examples/node_modules/requirejs/require.js"></script>
+  <script>require([
+      "../dist/pfe-clipboard.umd.js",
+      '../../pfe-band/dist/pfe-band.umd.js'
+    ])</script>
+  <style>
+    h2 {
+      margin-top: 20px;
+    }
+
+    h2 {
+      margin-bottom: 0;
+    }
+  </style>
+</head>
+
+<body unresolved>
+
+  <pfe-band color="lightest">
+    <h1 slot="pfe-band--header">&#x3C;pfe-element&#x3E;</h1>
+      <p>This element embeds a copy url button on your site. It also has a built in &#x3C;pfe-toast&#x3E; to notify the
+        user they successfully copied the url to their clipboard.</p>
+
+      <h2>Clipboard</h2>
+      <pfe-clipboard></pfe-clipboard>
+
+      <h2>Clipboard w/ iconhidden</h2>
+      <pfe-clipboard icon-hidden></pfe-clipboard>
+
+      <h2>Clipboard: w/ custom icon</h2>
+      <pfe-clipboard><span slot="pfe-clipboard--icon">⬇️</span></pfe-clipboard>
```

Comment:
One cool demo here would be to use `pfe-icon` to supply the custom icon.

Reply:
Yes, definitely!  That's a must.  I added pfe-icon to the demo.